paymentapi/src/main/java/fi/hel/verkkokauppa/payment/paytrail/PaytrailPaymentClient.java

```diff
@@ -67,6 +83,21 @@ public List<PaytrailPaymentMethod> getPaymentMethods(PaytrailPaymentContext cont
         }
     }
 
+    public PaytrailPaymentMitChargeSuccessResponse createMitCharge(PaytrailPaymentContext context, String paymentId, OrderWrapper orderWrapperDto, String token) throws ExecutionException, InterruptedException {
+        PaytrailClient paytrailClient = paytrailAuthClientFactory.createPaytrailClientFromPaymentContext(context);
+        CreateMitChargePayload payload = new CreateMitChargePayload();
+        BeanUtils.copyProperties(paymentPayloadConverter.convertToPayload(context, orderWrapperDto, paymentId), payload);
```

Comment:
Change this to use Mappers, example can be found here https://github.com/City-of-Helsinki/verkkokauppa-core/blob/464ade7d364ef1e9c4602d73c8935d80e9cf65f1/paymentapi/src/main/java/fi/hel/verkkokauppa/payment/mapper/PaymentMethodMapper.java